### PR TITLE
Remove github source

### PR DESCRIPTION
**Pull request by mohamedAlmukhtar**

### CODE REVIEW

**Comment by RogelioMart** [comment]
TL/DR
It looks great, there shouldn't be a problem when merging.

REPORT
Recurring event feature works great there are no issues with it. Also all other features seem to be unaffected by the new code, and the github source lines where deleted. Aside from some tests not working I do not believe that should affect the actual feature.

■ test/integration/events/user_visits_event.rb: 5 assertions, 1 failures, 1 errors
Error:
UserVisitsEvent#test_can_make_an_rsvp_for_event:
Capybara::Ambiguous: Ambiguous match, found 4 elements matching visible link or button nil with id RSVPsubmit
test/integration/events/user_visits_event.rb:56:in `block in <class:UserVisitsEvent>'
rails test test/integration/events/user_visits_event.rb:55

Failure:
UserVisitsEvent#test_should_see_the_correct_event_title [/home/v33mo/thinQStuff/mTest/tomatocan/test/integration/events/user_visits_event.rb:34]:
expected to find css "#calendar-event" but there were no matches
rails test test/integration/events/user_visits_event.rb:32

■ test/integration/users/home_page/guest_visits_home_page_test.rb: Doesn’t work

■ test/models/events_test.rb: 15 assertions, 1 failures, 0 errors
Failure:
EventTest#test_endat_greaterthan_startat [/home/user/thinQStuff/mTest/tomatocan/test/models/events_test.rb:97]:
Accepting events with start_at > end_at

■ test/models/merchandise_test.rb: 11 runs, 28 assertions, 0 failures

■ test/models/rsvpq_test.rb: 4 assertions, 0 failures, 0 errors

■ test/models/user_test.rb: 88 assertions, 0 failures, 3 errors
Error:
TestUser#test_get_youtube_id_test:
NoMethodError: undefined method `get_youtube_id' for #<User:0x0000562ffcbb61d8>
    test/models/user_test.rb:138:in `block in <class:TestUser>'
rails test test/models/user_test.rb:132

Error:
TestUser#test_videodesc_must_be_less_than255char:
ActiveModel::UnknownAttributeError: unknown attribute 'videodesc1' for User.
    test/models/user_test.rb:102:in `block (2 levels) in <class:TestUser>'
    test/models/user_test.rb:101:in `each'
    test/models/user_test.rb:101:in `block in <class:TestUser>'
rails test test/models/user_test.rb:100

Error:
TestUser#test_parse_youtube:
NoMethodError: undefined method `get_youtube_id' for #<User:0x0000562ffcdd8ec0>
    test/models/user_test.rb:128:in `block in <class:TestUser>'
rails test test/models/user_test.rb:124

To summarize the branch works well and I do not believe there would be an issue if this branch was merged.

**Comment by kmyee97** [comment]
Looks amazing. Tested it, and I believe there won't be any problems if the branch is merged. 

Report
The Github source is deleted. Excess HTML deleted. Features are not affecting the existing code. The only problem is the test files not working which won't affect the code. 


- test/integration/events/user_visits_event.rb 8 runs, 5 assertions, 1 failures, 1 errors, 0 skips

.E

Error:
UserVisitsEvent#test_can_make_an_rsvp_for_event:
Capybara::Ambiguous: Ambiguous match, found 4 elements matching visible link or button nil with id RSVPsubmit
    test/integration/events/user_visits_event.rb:56:in `block in <class:UserVisitsEvent>'


rails test test/integration/events/user_visits_event.rb:55

...F

Failure:
UserVisitsEvent#test_should_see_the_correct_event_title [/Users/kahlilyee/Documents/test/tomatocan/test/integration/events/user_visits_event.rb:34]:
expected to find css "#calendar-event" but there were no matches


rails test test/integration/events/user_visits_event.rb:32

**POTENTIAL SOLUTION: line:56  replace click_on id: "RSVPsubmit" with first(:id, "RSVPsubmit").click**
This will solve one of the errors for this test file.

- test/integration/users/home_page/guest_visits_home_page_test.rb

syntax error, unexpected `end', expecting end-of-input (SyntaxError)

**POTENTIAL SOLUTION:  delete lines 102 to 105. These lines do not test anything**
This will solve one of the errors for this test file.

- test/models/events_test.rb 11 runs, 15 assertions, 1 failures, 0 errors, 0 skips

...F

Failure:
EventTest#test_endat_greaterthan_startat [/Users/kahlilyee/Documents/test/tomatocan/test/models/events_test.rb:97]:
Accepting events with start_at > end_at


rails test test/models/events_test.rb:89

- test/models/merchandise_test.rb test/models/merchandise_test.rb 

- test/models/rsvpq_test.rb: 4 assertions, 0 failures, 0 errors

- test/models/user_test.rb: 88 assertions, 0 failures, 3 errors

.......E

Error:
TestUser#test_videodesc_must_be_less_than255char:
ActiveModel::UnknownAttributeError: unknown attribute 'videodesc1' for User.
    test/models/user_test.rb:102:in `block (2 levels) in <class:TestUser>'
    test/models/user_test.rb:101:in `each'
    test/models/user_test.rb:101:in `block in <class:TestUser>'


rails test test/models/user_test.rb:100

..E

Error:
TestUser#test_parse_youtube:
NoMethodError: undefined method `get_youtube_id' for #<User:0x00007fb75cbc1a68>
    test/models/user_test.rb:128:in `block in <class:TestUser>'


rails test test/models/user_test.rb:124

...........E

Error:
TestUser#test_get_youtube_id_test:
NoMethodError: undefined method `get_youtube_id' for #<User:0x00007fb75c7431a8>
    test/models/user_test.rb:138:in `block in <class:TestUser>'


rails test test/models/user_test.rb:132


**tl/dr The branch could merge with no problems since the errors are in the test files. I listed 2 potential solutions that will clean some errors.**
